ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -1147,7 +1181,7 @@ open class TableViewCell: UITableViewCell {
         if let trailingAccessoryView = trailingAccessoryView {
             let yOffset = UIScreen.main.roundToDevicePixels(topOffset + (labelSize.height - trailingAccessoryViewSize.height) / 2)
             let availableWidth = textAreaWidth - labelSize.width - leadingAccessoryAreaWidth
-            let leadingMargin = TableViewCell.labelTrailingAccessoryMarginLeading(text: text)
+            let leadingMargin = TableViewCell.labelTrailingAccessoryMarginLeading(text: text, labelAccessoryViewMarginLeading: tokens.labelAccessoryViewMarginLeading)
```

Comment:
nit: can you break these long lines into multiple ones (one for each param)?
(Here and throughout the changes being made.)

Reply:
Was this one updated? I still see all in one line. (some other instances in this PR as well)